packages/saber-plugin-feed/lib/utils.js

```diff
@@ -26,3 +26,33 @@ exports.getFeedPath = (feedPath, defaultPath) => {
 exports.resolveURL = (base, pathname) => {
   return new URL(pathname, base).href
 }
+
+/**
+ * Removes XML-invalid characters from a string.
+ * @param {string} string - a string potentially containing XML-invalid characters, such as non-UTF8 characters, STX, EOX and so on.
+ * @param {boolean} removeDiscouragedChars - a string potentially containing XML-invalid characters, such as non-UTF8 characters, STX, EOX and so on.
+ * @return : a sanitized string without all the XML-invalid characters.
+ */
```

Comment:
Can you add a reference to those RegExp?

Reply:
updated